src/reference/antora/modules/ROOT/pages/whats-new.adoc

```diff
@@ -79,6 +79,13 @@ The `AbstractRecentFileListFilter` strategy has been introduced to accept only t
 The respective implementations are provided: `RecentFileListFilter`, `FtpRecentFileListFilter`, `SftpRecentFileListFilter` and `SmbRecentFileListFilter`.
 See xref:file/reading.adoc[Reading Files] for more information.
 
+[[x6.5-file-exists-mode-expression]]
+== FileExistsMode Expression Support
+
+The remote file gateways (`AbstractRemoteFileOutboundGateway`) now support dynamic resolution of `FileExistsMode` at runtime via SpEL expressions. This allows determining the action to take when files already exist based on message content or other conditions.
```

Comment:
This is a `What's New` chapter.
Just thesis extracts for the rest the docs.
So, there is just enough to say one sentence, like your first one, and then link to the target chapter.